sdk/pom.xml

```diff
@@ -157,6 +157,12 @@
       <artifactId>mockito-4-2_2.12</artifactId>
       <version>3.2.11.0</version>
     </dependency>
+    <!-- https://mvnrepository.com/artifact/org.apache.maven/maven-model -->
+    <dependency>
+      <groupId>org.apache.maven</groupId>
+      <artifactId>maven-model</artifactId>
+      <version>3.9.0</version>
+    </dependency>
```

Comment:
why do we need this dependency?

Reply:
We don't anymore - that was for reading the pom.xml